java/src/gov/usgs/processingformats/TravelTimeSession.java

```diff
@@ -0,0 +1,609 @@
+package gov.usgs.processingformats;
+
+import java.util.*;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import gov.usgs.processingformats.ProcessingInt;
+
+public class TravelTimeSession implements ProcessingInt {
```

Comment:
brief class description comment is helpful as you have done in most other classes